lib/world/sequential.rb

```diff
@@ -72,6 +72,13 @@ def initialize(datafile = nil)
 
         @city_populations = JSON.parse(datafile) if datafile
 
+        # import hospitals
+        file = File.read './config/healthcare_facilities.json'
+        providers = JSON.parse(file)
+        providers.each do |_provider_name, provider_stats|
```

Comment:
What is the point of this `each` loop?

You aren't actually doing anything with the `Synthea:Hospital` after you create it.